src/main/scala/emm/effects/Binder.scala

```diff
@@ -29,6 +30,22 @@ trait BinderLowPriorityImplicits {
       NAP.pack(back)
     }
   }
+
+  implicit def binderState[S, C <: Effects](implicit C: Binder[C], T: Traverser[C], NN: NonNested[C]): Binder[State[S, ?] |: C] = new Binder[State[S, ?] |: C] {
```

Comment:
This actually doesn't achieve what we wanted, since `State` lacks a `Traverse`, meaning that it can *only* be the outer-most effect with this implementation!  What we really want is something that satisfies the following:

```scala
(implicit NAP: NestedAtPoint[C, StateT, F, T], F: Binder[F], TB: Binder[T], TT: Traverser[T]): Binder[C]
```

Or something thereabouts.